Vagrantfile

```diff
@@ -0,0 +1,157 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure("2") do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  config.vm.box = "ubuntu/xenial64"
+
+  # Create a forwarded port mapping which allows access to a specific port
+  # within the machine from a port on the host machine and only allow access
+  # via 127.0.0.1 to disable public access
+  config.vm.network "forwarded_port", guest: 7474, host: 7474, host_ip: "127.0.0.1"
+  config.vm.network "forwarded_port", guest: 7473, host: 7473, host_ip: "127.0.0.1"
+  config.vm.network "forwarded_port", guest: 7472, host: 7472, host_ip: "127.0.0.1"
+
+  # Share an additional folder to the guest VM or configure . The first argument is
+  # the path on the host to the actual folder. The second argument is
+  # the path on the guest to mount the folder. And the optional third
+  # argument is a set of non-required options.
+
+  project_name = "neo4j"
+  fail "Vagrantfile project_name variable cannot be blank" if project_name.nil? || project_name == ""
+
+  config.vm.synced_folder ".", "/home/ubuntu/#{project_name}"
```

Comment:
On the host machine, the project folder could be located anywhere. Inside the VM, the project folder is located here. This config sym links the two.